.github/workflows/deploy-my-kibana.yml

```
@@ -17,14 +17,35 @@ on:
 
 permissions:
   contents: read
+  # if you listen for PRs, use this to use some comment reactions
+  pull-requests: write
 
 jobs:
+  trim-github-comment:
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/oblt-deploy') }}
```

We cannot use a different string function to search for regexin the if conditions. It requires a step instead.

Using this approach should avoid validating all the comments but just the ones with `/oblt-deploy`. It might be a good balance to support spaces in the comment but without running always.

---

We only need to listen for comments on PRs, that contains `/oblt-deploy` and are created by a user.


```suggestion
    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/oblt-deploy') && github.event.comment.user.type == 'User' }} 
```

---

put the faster conditions first to evaluate them first and exist from the ìf` fast

```suggestion
    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/oblt-deploy') }}
    if: ${{ github.event.issue.pull_request && github.event.comment.user.type == 'User' 
 && contains(github.event.comment.body, '/oblt-deploy') }}
```